### PR TITLE
Buttons components

### PR DESCRIPTION
**Pull request by BlackMG27**
Could you take a look at the hover effect? I'm still stumped on that part. Thanks!

### CODE REVIEW

**Comment by peteherbst** [comment]
@BlackMG27 I'll see what I can do about the hover effect

File: src/components/buttons/ButtonPrimary.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonPrimary.tsx:1]: - Make sure you have the relevant VS Code extensions installed, most of these issues were displayed in VS Code (e.g. ESLint, Prettier, Tailwind CSS IntelliSense)
- className properties should all be on 1 line (we should be consistent with this)
- Remove semi-colons (for consistency)
- Font-family should be `Graphik, sans-serif` per the style guide
- in the `button` element, `text-white` and `text-fieldGrayLighter` apply the same CSS properties (cssConflict), choose one, not both
- `import localFont from "next/font/local" is declared but never read, remove it

File: src/components/buttons/ButtonsSecondary.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonsSecondary.tsx:1]: - className properties should all be on 1 line (we should be consistent with this)
- class `text-fieldGrayLigher` is misspelled, it should be `text-fieldGrayLighter`
- Font-family should be `Graphik, sans-serif` per the style guide
- add a blank line at the end of the file

File: src/components/buttons/ButtonLarge.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonLarge.tsx:1]: - className properties should all be on 1 line (we should be consistent with this)
- Font-family should be `Graphik, sans-serif` per the style guide
- add a blank line at the end of the file

**Review by peteherbst — COMMENTED** [review]
I've added comments, in the pull request, on the 3 files for the button components that are in: `src/components/buttons`.

Looking at the [style guide page](https://www.fieldmuseum.org/about/styleguide#buttons), we're also missing example components for:

- tertiary
- shuffle
- inverse button primary
- inverse button large
- inverse tertiary white

Please go back and make all of the changes I've indicated, add the missing components, and let me know when you've pushed your updates to the `buttons-components` branch. Your goal should be to have all of these updates done by Monday October 30.

**Comment by peteherbst** [comment]
I can look at the hover effect later, once you've finished the changes I've listed prior in this PR.

File: src/components/buttons/ButtonsSecondary.tsx
Comment (BlackMG27) [reply src/components/buttons/ButtonsSecondary.tsx:1]: Okay, thanks. I'll take a look.

File: src/components/buttons/ButtonPrimary.tsx
Comment (BlackMG27) [reply src/components/buttons/ButtonPrimary.tsx:1]: Okay. I'll make the necessary corrections.

**Comment by BlackMG27** [comment]
@peteherbst Okay, thanks!

File: src/components/buttons/ButtonsSecondary.tsx
Comment (peteherbst) [reply src/components/buttons/ButtonsSecondary.tsx:1]: - The blue border on the secondary button looks thicker in our compared to the style guide, can you reduce the border size?
- It looks like the style guide button has more horizontal padding, please increase

File: src/components/buttons/ButtonPrimary.tsx
Comment (peteherbst) [reply src/components/buttons/ButtonPrimary.tsx:1]: - It looks like the style guide button has more horizontal padding, please double-check

File: src/components/buttons/ButtonLarge.tsx
Comment (peteherbst) [reply src/components/buttons/ButtonLarge.tsx:1]: - It looks like the style guide button has more horizontal padding, please increase

File: src/components/buttons/ButtonShuffle.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonShuffle.tsx:1]: - The gray color that appears with the hover effect should include the shuffle icon, compare with the style guide page

File: src/components/buttons/ButtonInversePrimary.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonInversePrimary.tsx:1]: - Increase horizontal padding; compare to style guide

File: src/components/buttons/ButtonInverseLarge.tsx
Comment (peteherbst) [inline src/components/buttons/ButtonInverseLarge.tsx:1]: - Increase horizontal padding; compare to style guide

**Comment by peteherbst** [comment]
> Could you take a look at the hover effect? I'm still stumped on that part. Thanks!

We should stick to the built-in Tailwind classes as much as possible, rather than breaking out the CSS into a separate module.scss file. If we have to do that, we can, but for now can you please take a look at this documentation and see if you can adapt it for our button transitions? It looks like this provides what we need.

https://tailwindcss.com/docs/transition-property

**Pull request merged**